scripts/release.sh

```diff
@@ -23,6 +23,8 @@ fi
 release_tag="$1"
 # without the starting 'v'
 release_version=$(echo "$release_tag" | cut -c2-)
+# v1 from v1.2.3
```

Comment:
I don't think that v2 will appear any time soon 😅

Reply:
Hopefully never but I'm getting push back from others when trying to use `latest` 🤷